core/tests/protractor_utils/general.js

```diff
@@ -42,6 +42,11 @@ var checkForConsoleErrors = function(errorsToIgnore) {
       });
     }
 
+    // Temporary change for AngularJS Migration.
+    browserLogs = browserLogs.filter(function(browserLog) {
```

Comment:
Hi @seanlip & @apb7, this error logs unexpected i am unable to reproduce this in my machine.
can you please pull the branch and run e2e for additionalEditorFeatures ? and see if this is logged Thanks!

Reply:
Please dont forgot to removed the above temporary change before running e2e for additionalEditorFeatures

Reply:
This is resolved Thanks!